sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/generators.scala

```diff
@@ -343,6 +343,63 @@ abstract class ExplodeBase extends UnaryExpression with CollectionGenerator with
   }
 }
 
+@ExpressionDescription(
+  usage = "_FUNC_(expr) - Separates the elements of array `expr` into multiple rows recursively.",
+  examples = """
+    Examples:
+      > SELECT _FUNC_(array(10, 20));
+       10
+       20
+      > SELECT _FUNC_(a) FROM VALUES (array(1,2)), (array(3,4)) AS v1(a);
+       1
+       2
+       3
+       4
```

Comment:
@yaooqinn given this examples, yes looks equivalent. How are they different in this PR?

```scala
scala> sql("select unnest(a) FROM VALUES (array(1,2)), (array(3,4)) AS v1(a)").show()
+------+
|unnest|
+------+
|     1|
|     2|
|     3|
|     4|
+------+


scala> sql("select unnest(array(10, 20))").show()
+------+
|unnest|
+------+
|    10|
|    20|
+------+


scala> sql("select explode(a) FROM VALUES (array(1,2)), (array(3,4)) AS v1(a)").show()
+---+
|col|
+---+
|  1|
|  2|
|  3|
|  4|
+---+


scala> sql("select explode(array(10, 20))").show()
+---+
|col|
+---+
| 10|
| 20|
+---+
```

Reply:
Sorry，the examples are misleading，should use multidimensional arrays i.e array(array(1,2),array(3,4)). Then it will show the difference between unnest and explode. I
 will add some new examples here.